### ISSUE TITLE
bug: MPC-TLS backend deadlocks when receiving no data prior to connection close

### ISSUE THREAD

**Issue by sinui0** [tlsnotary/tlsn#626]

**Comment by themighty1** [comment]
I dont believe this is an issue anymore.  I tried to close the connection abruptly on the server side after the handshake both before/after the prover sent the request. The prover exited gracefully.
I think you can close this @sinui0 , unless you have sth ele in mind.
